bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/ElectionRecord/ElectionConstants.cs

```diff
@@ -0,0 +1,34 @@
+using ElectionGuard.ElectionSetup;
+
+namespace ElectionGuard.Decryption.ElectionRecord;
+
+/// <summary>
+/// Serializable election constants used for a specific election
+/// </summary>
+public record ElectionConstants : DisposableRecordBase
```

Comment:
`ElectionConstants` are the constants passed in as part of the election record, which may be different from the `Constants` that are compiled in the codebase.